lib/avo/fields/file_field.rb

```diff
@@ -24,6 +24,17 @@ def initialize(id, **args, &block)
       def path
         rails_blob_url value
       end
+
+      def value
+        final_value = super
+        # On edit view always show the persisted image. Related: issue#3008
+        if final_value.instance_of?(ActiveStorage::Attached::One) && final_value.image? && view == "edit"
```

Comment:
> `final_value.image?`

Should we apply this only to images? I think we should remove that guard and apply this behavior to all file types, WDYT?

Reply:
Agreed. I'll look into it